website/core/Footer.js

```diff
@@ -27,35 +27,35 @@ class Footer extends React.Component {
               href={
                 this.props.config.baseUrl +
                 'docs/' +
-                this.props.language +
-                '/getting-started.html'
+                // this.props.language +
```

Comment:
we should leave this in. it produces a slightly weird url right now but it still works, and we'll add translations shortly

Reply:
Okay. Reverted back the Footer.js